src/Form/Form.js

```diff
@@ -87,11 +87,31 @@ class Form {
 
         this.categorizeInputs()
 
+        this.logFormInfo()
+
         if (shouldAutoprompt) {
             this.promptLoginIfNeeded()
         }
     }
 
+    logFormInfo () {
+        if (!shouldLog()) return
+
+        console.log(`Form type: %c${this.getFormType()} ${this.formAnalyzer.autofillSignal}`, 'font-weight: bold')
```

Comment:
Also, would use of `console.group` (similar to `logMatching`) make sense to use here?

Reply:
The `%c` bit is interpolated in the console with the next parameter, in this case the bolding style. See https://developer.chrome.com/docs/devtools/console/format-style/.

The grouping here is less useful because we won't be logging forms one after the other (there are the field logs in between).